static/js/components/TelescopeMap.jsx

```diff
@@ -0,0 +1,195 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { useDispatch } from "react-redux";
+import {
+  ComposableMap,
+  Geographies,
+  Geography,
+  Marker,
+  useZoomPan,
+} from "react-simple-maps";
+
+import world_map from "../../images/maps/world-110m.json";
```

Comment:
Is this committed now somewhere?

Reply:
Yes ! Forgot it in the first commit hahaha. I noticed something, the map doesnt show up in smaller screens. Maybe i could put the normal list back for smaller screens only, otherwise find a way to fix that on the map

Reply:
@Theodlz I.e. the behavior is different for the local vs online loading?

Reply:
Not at all ! Maybe my last message isnt clear enough. Replacing the link by the file itself works perfectly, no issue on that. I was mentionning another issue of how it doesnt display on mobile or small screens, but that has nothing to do with replacing the link by the file directly :).

Reply:
Ah got ya. There are other places in the app where desktop vs mobile experience is different, so could just lean into that and explicitly only call the map on desktop and otherwise just show the list form?

Reply:
@mcoughlin  done ! Just used the previous version (simple list) on small screens, and the map on bigger devices. Strange thing, but that issue of the map disappearing only occurs on google chrome, the map resizes and works perfectly on firefox...